### DISCUSSION TITLE
General Discussion

### DISCUSSION THREAD

**Discussion by admin** [vs-ude/mic-1-toolbox#120] [General]
Welcome to the MIC-1-toolbox! Here you can ask questions, get help, share your experience. 
If you have suggestions, feel free to create your own discussion.
If you are new to GitHub Discussions, you can have a look at the [official guide](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion) to learn how to utilize them.

**Comment by TheRustyStorm** [comment]
Bugreport: 
(0x60)POWER: OPC = TOS
MDR=0
H=TOS

Animation ist buggy bei H=TOS, animate checkbox wurde vorher getoggled

**Reply by OskarCarl** [reply]
Ref #215
